src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Action/ActionCardAction.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action;
+
+use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Input\InputInterface;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ *
+ * @see https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference#actioncard-action
+ */
+class ActionCardAction implements ActionInterface
+{
+    protected $options = [];
```

Comment:
```suggestion
    protected $options = ['@type' => 'ActionCard'];
```
and remove the business logic from `toArray` ?

Reply:
works for me 👍

Reply:
Done